### ISSUE TITLE
Validation loss less than train loss

### ISSUE THREAD

**Issue by Deepthi-vr-code** [MIC-DKFZ/nnUNet#2715]
Hi,
I am implementing nnUNet with small dataset(88 images), and is overfitting. So I augmented the dataset to 880 and trained again. Now plot is ok, but validation loss is less than train loss. Could you please explain why? is it an issue? Attaching both plots

![Image](https://github.com/user-attachments/assets/a6384214-fb57-47ea-bd1c-a1d164402669)

![Image](https://github.com/user-attachments/assets/f4ca24b3-f8a8-46e9-b25a-c28e4c478339)

**Comment by Deepthi-vr-code** [comment]
Is this due to default augmentation applied for training dataset and not to validation set in nnUNet? But for other dataset there is no issue. Is this an issue? By changing initial learning rate or weight decay can i modify this?

**Comment by Karol-G** [comment]
Hey,

Is this issue still relevant? Otherwise, I will close it in 14 days.

Best,
Karol

**Comment by Karol-G** [comment]
Closing this issue due to inactivity.
